assets/js/pages/Profile/ProfileForm.jsx

```diff
@@ -180,6 +184,38 @@ function ProfileForm({
               disabled
             />
           </div>
+          {analyticsEnabledConfig && (
+            <Label className="col-start-1 col-span-2 pt-2">
+              Analytics Opt-in
+            </Label>
+          )}
+          {analyticsEnabledConfig && (
+            <div className="col-start-3 col-span-4">
+              <div className="pt-2">
+                <div className="flex items-center">
+                  <Input
+                    className="mr-2"
+                    type="checkbox"
+                    role="checkbox"
+                    checked={analyticsEnabledState}
+                    onChange={() => setAnalyticsState(!analyticsEnabledState)}
```

Comment:
This `onChange` usually is done this way:
```
onChange={(value) => setAnalyticsState(value)}

Reply:
I tried this again now, and I can't get it to work with this Input component. 😅